src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs

```diff
@@ -0,0 +1,226 @@
+
+                    Assert.False(string.IsNullOrWhiteSpace(storageKey), "Environment variable TEST_STORAGE_KEY has not been set.  Set it to storage account key.");
+                    Assert.False(string.IsNullOrWhiteSpace(importBacpacContainer), "Environment variable TEST_IMPORT_CONTAINER has not been set.  Set it to a valid storage container URL.");
+                }
+
+                // Export database to bacpac
+                sqlClient.Databases.Export(resourceGroup.Name, server.Name, dbName, new ExportRequestParameters()
+                {
+                    AdministratorLogin = login,
+                    AdministratorLoginPassword = password,
+                    AuthenticationType = AuthenticationType.SQL,
+                    StorageKey = storageKey,
+                    StorageKeyType = StorageKeyType.StorageAccessKey,
+                    StorageUri = exportBacpacLink
+                });
+
+                // Import bacpac to existing database
+                ImportExportOperationResponse importResponse = sqlClient.Databases.Import(resourceGroup.Name, server.Name, dbName2, new ImportExtensionRequestParameters()
+                {
+                    AdministratorLogin = login,
+                    AdministratorLoginPassword = password,
+                    AuthenticationType = AuthenticationType.SQL,
+                    StorageKey = storageKey,
+                    StorageKeyType = StorageKeyType.StorageAccessKey,
+                    StorageUri = exportBacpacLink
+                });
+
+                // Get Database import/export operation
+                ImportExportOperationResponse importStatusResponse = sqlClient.ImportExportOperations.GetByDatabase(resourceGroup.Name, server.Name, dbName2, importResponse.RequestId.Value);
+                
+                // List Server import/export operations
+                IEnumerable<ImportExportOperationResponse> listOperationResponse = sqlClient.ImportExportOperations.ListByServer(resourceGroup.Name, server.Name);
+                Assert.Equal(2, listOperationResponse.Count());
+                Assert.Equal(1, listOperationResponse.Count(o => o.RequestType.Equals("Export") && o.DatabaseName.Equals(dbName)));
+                Assert.Equal(1, listOperationResponse.Count(o => o.RequestType.Equals("Import") && o.DatabaseName.Equals(dbName2)));
+
+                // Get Server import/export operation
+                foreach(ImportExportOperationResponse operation in listOperationResponse)
+                {
+                    ImportExportOperationResponse getOperationResponse = sqlClient.ImportExportOperations.GetByServer(resourceGroup.Name, server.Name, operation.RequestId.Value);
+                    Assert.NotNull(getOperationResponse);
+                }
+            });
+        }
+
+        [Fact]
+        public void TestExportImportNewDatabaseGetOperation()
```

Comment:
these 2 tests are basically the same, right? if it reduces a lot of duplication can you have these 2 test methods call one common method that gets called with 2 different modes, e.g. `TestExportImportGetOperation(bool existingDatabase)`?